hbf/rescrape/laundry.py

```diff
@@ -51,6 +45,12 @@ def laundry(dirtyLaundry):
     dirtyLaundry["Year Built"] = dirtyLaundry["Year Built"].astype(float)
     dirtyLaundry["Lot Size"] = dirtyLaundry["Lot Size"].astype(float)
 
+    # Log clean SoldHomeData
+    logToFile(__name__, dirtyLaundry, 'INFO')
+    # Log clean SoldHomeData datatypes
+    logToFile(__name__, dirtyLaundry.dtypes, 'INFO')
+
     print(dirtyLaundry)
-    print(dirtyLaundry.dtypes)
```

Comment:
this was literally just a debugging/test statement @AmritPatel .

Reply:
That is exactly what the log file is for!